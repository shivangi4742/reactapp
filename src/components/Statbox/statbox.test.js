import React from 'react';
import { shallow } from 'enzyme';
import StatBox from './index';
import  toJSON from 'enzyme-to-json';

describe('StatBox', () => {
    it('matches the snapshots of StatBox component', () => {
        const tree = shallow(<StatBox />);
        expect(toJSON(tree)).toMatchSnapshot();
    });
})