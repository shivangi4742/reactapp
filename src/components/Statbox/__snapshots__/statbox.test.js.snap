// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatBox matches the snapshots of StatBox component 1`] = `<styled.div />`;
